.github/workflows/release.yml

```diff
@@ -122,55 +19,37 @@ jobs:
           echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
           nix-shell --pure --run "echo" ./shell.nix
 
-      - name: Check if the chart is publishable
-        run: |
-          TAG=${{ github.event.release.tag_name }}
-          nix-shell --pure --run "./scripts/update-chart-version.sh --tag $TAG --publish-release" ./shell.nix
-
       - name: Set Image Org
         run: |
-          [ -z "${{ secrets.IMAGE_ORG }}" ] && IMAGE_ORG=openebs || IMAGE_ORG=${{ secrets.IMAGE_ORG}}
+          [ -z "${{ secrets.IMAGE_ORG }}" ] && IMAGE_ORG=openebs || IMAGE_ORG=${{ secrets.IMAGE_ORG }}
           echo "IMAGE_ORG=${IMAGE_ORG}" >> $GITHUB_ENV
 
-      - name: Set IMAGE_TAG and BRANCH
-        run: |
-          BRANCH=${{ github.ref_name }}
-          echo "BRANCH=$BRANCH" >> $GITHUB_ENV
-          echo "IMAGE_TAG=$(awk -F': ' '/^version:/ {print $$2}' deploy/helm/charts/Chart.yaml)" >> $GITHUB_ENV
+      - name: Log in to GHCR for Helm
+        run: nix-shell --pure --run "echo ${{ secrets.GITHUB_TOKEN }} | helm registry login ghcr.io -u ${{ github.actor }} --password-stdin" ./shell.nix
 
-      - name: Set Build Date
-        id: date
+      - name: Pull OCI chart from GHCR
+        env:
+          IMAGE_ORG: ${{ env.IMAGE_ORG }}
+          RAW_TAG: ${{ github.ref_name }}
         run: |
-          echo "DATE=$(date -u +'%Y-%m-%dT%H:%M:%S%Z')" >> $GITHUB_OUTPUT
+          TAG="${RAW_TAG#v}"
+          echo "Using chart version: $TAG"
 
-      - name: Docker meta
-        id: docker_meta
-        uses: docker/metadata-action@v4
-        with:
-          # add each registry to which the image needs to be pushed here
-          images: |
-            ${{ env.IMAGE_ORG }}/zfs-driver
-            quay.io/${{ env.IMAGE_ORG }}/zfs-driver
-            ghcr.io/${{ env.IMAGE_ORG }}/zfs-driver
-          tags: |
-            type=raw,value=latest,enable=false
-            type=raw,value=${{ env.IMAGE_TAG }}
-
-      - name: Print Tag info
-        run: |
-          echo "BRANCH: ${{ env.BRANCH }}"
-          echo "${{ steps.docker_meta.outputs.tags }}"
+          rm -rf deploy/helm/charts
```

Comment:
why is this being deleted?

Reply:
It's a safety measure to ensure the action doesn't confuse with other charts in the workspace.